packages/components/package.json

```diff
@@ -36,21 +36,30 @@
   },
   "dependencies": {
     "ally.js": "^1.4.1",
+    "classnames": "^2.2.5",
     "date-fns": "^1.27.2",
+    "d3-shape": "^1.2.0",
+    "focus-outline-manager": "^1.0.2",
     "immutable": "^3.8.1",
+    "invariant": "^2.2.2",
+    "keycode": "^2.2.0",
     "lodash": "^4.17.4",
     "memoize-one": "^5.0.0",
     "rc-slider": "8.6.3",
     "rc-tooltip": "3.7.3",
     "react-autowhatever": "10.2.0",
+    "react-bootstrap": "0.32.4",
+    "react-css-transition": "^0.7.4",
```

Comment:
Would be great to drop this one in favor of react-transition-group ( but not to do in this PR )

Reply:
Yes it would be great